osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -19,11 +19,13 @@ public class LayoutSettings : SettingsSubsection
         private FillFlowContainer letterboxSettings;
 
         private Bindable<bool> letterboxing;
+        private Bindable<Size> sizeFullscreen;
```

Comment:
Any reason for using `Bindable<Size>` here over `BindableSize`?

Reply:
`config.GetBindable<Size>(FrameworkSetting.SizeFullscreen)` call

Reply:
Oh, alright. You could always use `config.BindWith` instead, if you want to.